Exoft.Gamification.Api.Common/Helpers/AutoMapper.cs

```diff
@@ -15,31 +15,11 @@ public AutoMapper()
             // map to model
 
             CreateMap<User, UserModel>()
-                .ForMember(i => i.Roles,
-                                m => m.MapFrom(
-                                    p => p.Roles.Select(d => new RoleModel()
-                                    {
-                                        Id = d.Role.Id,
-                                        Text = d.Role.Text
-                                    }).ToList()))
-                .ForMember(i => i.Achievements,
-                                m => m.MapFrom(
-                                    p => p.Achievements.Select(d => new AchievementModel()
-                                    {
-                                        Id = d.Achievement.Id,
-                                        Description = d.Achievement.Description,
-                                        Icon = d.Achievement.Icon,
-                                        Name = d.Achievement.Name,
-                                        XP = d.Achievement.XP
-                                    }).ToList()));
+                .ForMember(i => i.AvatarId, m => m.MapFrom(p => p.Avatar.Id));
```

Comment:
It's possible for user to not have icon.... So it might cause null reference exception